packages/rpc-transformers/src/response-transformer-allowed-numeric-values.ts

```diff
@@ -37,3 +37,19 @@ export const jsonParsedAccountsConfigs = [
     ['data', 'parsed', 'info', 'commission'],
     ['data', 'parsed', 'info', 'votes', KEYPATH_WILDCARD, 'confirmationCount'],
 ];
+export const innerInstructionsConfigs = [
+    ['index'],
+    ['instructions', KEYPATH_WILDCARD, 'accounts', KEYPATH_WILDCARD],
+    ['instructions', KEYPATH_WILDCARD, 'programIdIndex'],
+    ['instructions', KEYPATH_WILDCARD, 'stackHeight'],
+];
+export const messageConfig = [
+    ['addressTableLookups', KEYPATH_WILDCARD, 'writableIndexes', KEYPATH_WILDCARD],
+    ['addressTableLookups', KEYPATH_WILDCARD, 'readonlyIndexes', KEYPATH_WILDCARD],
+    ['header', 'numReadonlySignedAccounts'],
+    ['header', 'numReadonlyUnsignedAccounts'],
+    ['header', 'numRequiredSignatures'],
+    ['instructions', KEYPATH_WILDCARD, 'accounts', KEYPATH_WILDCARD],
+    ['instructions', KEYPATH_WILDCARD, 'programIdIndex'],
+    ['instructions', KEYPATH_WILDCARD, 'stackHeight'],
```

Comment:
Would have DRY'd these up too, but using `foo.map()` in this file at the top level would break tree-shakability.